framework/src/components/storage/entities/transaction.js

```diff
@@ -144,6 +144,10 @@ class Transaction extends BaseEntity {
 			fieldName: 'trs.rowId',
 		});
 
+		this.addField('dapp_name', 'string', {
```

Comment:
Is this necessary? If so, please combine with `addFilter` in https://github.com/LiskHQ/lisk/blob/790e9e9ba6ac378c224c0522a56fd1e1165862e4/framework/src/components/storage/entities/transaction.js#L237